app/view/home/controlButtons.js

```diff
@@ -624,17 +624,18 @@ getCurrentDisplayModeClass: function() {
             OkBtn: SDL.Button.create({
                 elementId: 'OK',
                 classNames: 'OkBtn',
+                classNameBindings: 'SDL.States.media.active:media_button',
                 time: 0,
-                presetName: 'OK',
+                presetName: 'PLAY_PAUSE',
                 actionDown: function() {
-
                   this._super();
-                  SDL.SDLController.onSoftButtonOkActionDown(this.presetName);
+                  const preset_actual_name = SDL.States.media.active ? this.presetName : 'OK';
```

Comment:
```suggestion
                  const preset_actual_name = SDL.States.media.active ? 'PLAY_PAUSE' : 'OK';
```
what do you think of having both values in the ternary? since it is always conditional is there any reason to have the `presetName` prop on this button?

Reply:
@iCollin I think it can we can remove `presetName` and replace it with a common function which can be used by both events of that button. Please take a look at https://github.com/smartdevicelink/sdl_hmi/pull/333/commits/c031858cedbd46e0b48e697575cc7ba52c11e69f